terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/test/java/org/terasoluna/gfw/web/pagination/PaginationTagTest.java

```diff
@@ -82,7 +86,77 @@ public void tearDown() throws Exception {
 
     @Test
     public void testRelease01() throws Exception {
+
         tag.release();
+
+        Field page = ReflectionUtils.findField(PaginationTag.class, "page");
+        ReflectionUtils.makeAccessible(page);
+        Field pathTmpl = ReflectionUtils.findField(PaginationTag.class,
+                "pathTmpl");
```

Comment:
You can use `ReflectionTestUtils#getField(Object targetObject, String name)` more easily.

Reply:
Implemented as explained.
c8f7b4e4e2f0659c6df5071dd78518f25e18c303